src/config/graphical-session/graphics-drivers/intel.md

```diff
@@ -24,3 +24,14 @@ environment variable `LIBVA_DRIVER_NAME`:
 |----------------------|-------------------|--------------------------|
 | `libva-intel-driver` | up to Coffee Lake | `LIBVA_DRIVER_NAME=i965` |
 | `intel-media-driver` | from Broadwell    | `LIBVA_DRIVER_NAME=iHD`  |
+
+## Troubleshooting
+
+The kernels packaged by Void are configured with
+`CONFIG_INTEL_IOMMU_DEFAULT_ON=y`, which can lead to issues with their graphics
+drivers, as reported by the [kernel
+documentation](https://www.kernel.org/doc/html/latest/x86/intel-iommu.html#graphics-problems).
+To fix this, it is necessary to disable IOMMU for the integrated GPU. This can
+be done by adding `intel_iommu=igfx_off` to your [kernel
```

Comment:
I have only noticed this workaround for Intel Generation 5 (Broadwell).

Reply:
Mine (Skylake - don't remember the graphics gen) locks up, but hasn't done so again since I switched on guc firmware. It's hard to determine where the issues come from.

How many devices do you deal with? Do you think your experience is anecdotal or the expected one? The kernel docs didn't suggest any particular gen.

Reply:
Well, it is broken _in the hardware_ for Broadwell, meaning you will have to enable this if you have 5th gen. For other versions, it could be firmware or something else, so I'm not sure.

Reply:
I didn't have problems with gen 7 and gen 3, but you're right this is all anecdotal on my side.

Reply:
https://www.phoronix.com/scan.php?page=news_item&px=Intel-IOMMU-Gfx-Default-Try

Reply:
I left it as a generic option, but which is recommened for Broadwell. What do you think?

Reply:
Seems reasonable. Maybe mention that you need `linux-firmware-intel`, somewhere on this page.

Reply:
It's a dependency for the `linux` and `linux-lts` packages. Do you think it needs to be?

Reply:
Fair enough

Reply:
Ended up adding it :p Thanks for the suggestion!